### PR TITLE
NativeAOT and C/C++ wrapper

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
That seems to be possible. And it's awesome.
image

### CODE REVIEW

**Comment by WhiteBlackGoose** [comment]
It's dynamically linked now. Should be statically

**Comment by WhiteBlackGoose** [comment]
🥳 
image

File: Sources/Samples/CPlusPlusPlayground/CPlusPlusPlayground.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//
+
+#include "CPlusPlusPlayground.h"
+#include "../../Wrappers/AngouriMath.CPP.Importing/AngouriMath.CPP.h"
+
+using namespace std;
+
+int main()
+{
+	Entity expr = "sin(x) + tan(x)^2 + sqrt(a x) +";
```
Comment (Happypig375) [inline Sources/Samples/CPlusPlusPlayground/CPlusPlusPlayground.cpp:11]: This isn't a valid expression?

File: Sources/Wrappers/AngouriMath.CPP.Exporting/Exceptions.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AngouriMath.CPP.Exporting
+{
+    public abstract class ObjectStorateException : Exception { }
```
Comment (Happypig375) [inline Sources/Wrappers/AngouriMath.CPP.Exporting/Exceptions.cs:7]: What's a storate?

File: Sources/Samples/CPlusPlusPlayground/CPlusPlusPlayground.cpp (hunk 1 above)
Comment (WhiteBlackGoose) [reply Sources/Samples/CPlusPlusPlayground/CPlusPlusPlayground.cpp:11]: It's not. I was testing exceptions :)

File: Sources/Wrappers/AngouriMath.CPP.Exporting/Exceptions.cs (hunk 2 above)
Comment (WhiteBlackGoose) [reply Sources/Wrappers/AngouriMath.CPP.Exporting/Exceptions.cs:7]: 🦆

**Pull request merged**